linkerd/examples/k8s.yaml

```diff
@@ -0,0 +1,19 @@
+admin:
+  port: 9990
+
+namers:
+- kind: io.l5d.experimental.k8s
+  host: localhost
+  port: 443
+  tls: true
+  tlsWithoutValidation: true
+  authTokenFile: linkerd/examples/io.l5d.k8s/kube_token
```

Comment:
I found this was the easiest/fastest way to spin up a linkerd that used the k8s namer

`./sbt linkerd-examples/k8s:run`

I figured this might be useful to others as well.  I'm not super attached to it, though.